src/test/regress/expected/materialized_view.out

```diff
@@ -200,3 +200,74 @@ SELECT count(*) FROM materialized_view;
 DROP MATERIALIZED VIEW materialized_view;
 DROP SCHEMA materialized_view CASCADE;
 NOTICE:  drop cascades to view air_shipped_lineitems
+-- modify statements on/with views
```

Comment:
I think `rewriteTargetView` is a good resource to check what to test. Some examples:

- [ ] RETURNING
- [ ] ON CONFLICT DO UPDATE .. SET ..
- [ ] DELETE/UPDATE/INSERT to the view itself
- [ ]  Junk columns (`resjunk` in the target list)
- [ ] subqueries in the WHERE clause

can you make sure to add tests for the above as well?

Reply:
In commits 2b8e10a1c4087455878805a7528025e8eaacd0cf and f214815e3045e70c8f6dc6c6b401cabedbd14da2, I added tests for these items